src/compiler/mod.rs

```diff
@@ -696,6 +725,7 @@ impl Compiler {
                 Value::Null => unreachable!(),
                 Value::String(_) => todo!(),
                 Value::Function { .. } => (),
+                Value::Break => todo!(),
```

Comment:
there is another exception macro that would fix more nicely here - `unreachable!()`